README.md

```diff
@@ -284,3 +285,32 @@ Finally, we run and wait for review.
 ```sh
 git pull
 ```
+=======
+## Main Quest: Moving from a JavaScript file to TypeScript
+
+our first step here is using a package manager(in our case Yarn) to install:
+```
+yarn add typescript
+```
+
+now we're going to rename our .js file to .ts using the following commands:
+
+```
+mv program_name.js program_name.ts
+```
+In the program we just renamed above we'll have to replace:
+``` const express = require('express') ``` for ```import express from 'express';```
+
+Lastly we're going to run this sequence of commands to finalize the setup:
```

Comment:
Will this render as if it's in the code block? Try to get VS Code to give you a preview.

Reply:
fixed